packages/shared/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@chromaui/shared-e2e",
+  "private": true,
+  "version": "0.0.1",
+  "description": "Shared Chromatic E2E",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/chromaui/test-archiver.git",
+    "directory": "packages/shared"
+  },
+  "author": "Chromatic <tom@chromatic.com>",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "files": [
+    "dist/**/*",
+    "README.md",
+    "*.js",
+    "*.d.ts"
+  ],
+  "scripts": {
+    "clean": "rimraf ./dist",
+    "prebuild": "yarn clean",
+    "build": "tsup",
+    "build:watch": "yarn build --watch",
+    "test:unit": "jest",
+    "lint": "eslint src/*",
+    "prettier": "prettier"
+  },
+  "devDependencies": {
+    "@auto-it/released": "^10.32.6",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
+    "@chromaui/archive-storybook": "~0.0.15",
+    "@jest/types": "^27.0.6",
+    "@playwright/test": "^1.39.0",
```

Comment:
What in the shared package uses `@playwright/test` still -- is it the Protocol Playwright types in the resource-archiver?

Reply:
Good catch. I'll double check but yes, I believe this is needed since we still reference playwright things. We can follow up with a fix for that.